src/index.js

```diff
@@ -1,54 +1,71 @@
 'use strict'
 
-const connect = require('pull-ws/client')
+const connect = require('it-ws/client')
 const mafmt = require('mafmt')
 const withIs = require('class-is')
-const Connection = require('interface-connection').Connection
-
 const toUri = require('multiaddr-to-uri')
-const debug = require('debug')
-const log = debug('libp2p:websockets:dialer')
-
+const log = require('debug')('libp2p:websockets:transport')
+const abortable = require('abortable-iterator')
 const createListener = require('./listener')
+const { AbortError } = abortable
```

Comment:
There's now an `AbortError` in interface-transport, should we use that one instead?